app/models/game_night.rb

```diff
@@ -10,19 +10,23 @@ class GameNight < ApplicationRecord
   validates :number_of_games, presence: true
   validates :name, presence: true
 
-  def attendees
```

Comment:
If you change this, can you pull in main and confirm that my changes to the game_night_serializer are still working? Running tests will confirm or deny it if you pull in main.

Reply:
There will be a conflict (or it might just keep it all in there so watch for that) as I changed the attendee part of the serializer and changed the test.

Reply:
sure pulling from right now

Reply:
just pulled from main and all test still passing

Reply:
Solid!